src/contract.rs

```diff
@@ -45,6 +45,11 @@ pub fn instantiate(
     };
     CONFIG.save(deps.storage, &config)?;
 
+    let off_chain_timestamps = OffChainTimestamps {
```

Comment:
Some variant of this code need to be present in migrateMsg in a backwards compatible fashion. On a migrate message, we aren't going to run instantiate.